modules/layers/src/icon-layer/icon-manager.js

```diff
@@ -278,7 +278,11 @@ export default class IconManager {
       if (!this._texture) {
         this._texture = new Texture2D(this.gl, {
           width: this._canvasWidth,
-          height: this._canvasHeight
+          height: this._canvasHeight,
+          parameters: {
```

Comment:
Make default texture parameters a constant that is shared between pre-packed and auto-packed